packages/db-objects-redis/src/lib/objects/objectsUtils.ts

```diff
@@ -615,14 +626,15 @@ export function checkObject(obj: ioBroker.Object, options: Record<string, any>,
     return true; // ALL OK
 }
 
+// FIXME: Return type?
```

Comment:
It feels like this should have an easy return type, but the control flow isn't clean. There's a mix of `return tools.maybeCallback...` on the root level and inside callbacks.

Reply:
To achieve 100 % clean handling we would need to make getUserGroup in db itself async or in utils and map to utils, but db is exposed so not sure if we should do it right now